### ISSUE TITLE
Back button in header tap target too small

### ISSUE THREAD

**Issue by dharding** [covid-alert-ny/covid-green-app#258]
I find myself frequently missing the tap target for the back navigation in the header throughout the app. Please ensure this tap target covers the entire area behind the label and arrow.

**Comment by JohnAGill** [comment]
This has been addressed and will be in the next build

**Comment by helloantoine** [comment]
Tried on iOS and Android. Looked better. (still hate the animation)
